### PR TITLE
Remove deprecated affine_scalar

### PR DESCRIPTION
**Pull request by giovp**
Replace deprecated `tfb.AffineScalar` with `tfb.Shift(shift=...)(tfb.Scale(scale=...))` for `tfb.MaskedAutoregressiveFlow` as suggested in #448 . 
Pytest is failing but I can't figure out why.

Sorry for the previous PR, had an issue with mismatching users w/ google CLA...

### CODE REVIEW

File: tensorflow_probability/python/bijectors/masked_autoregressive.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import tensorflow.compat.v1 as tf1
 import tensorflow.compat.v2 as tf
 
-from tensorflow_probability.python.bijectors import affine_scalar
+from tensorflow_probability.python.bijectors import shift, scale
```
Comment (srvasude) [inline tensorflow_probability/python/bijectors/masked_autoregressive.py]: Note that shift is also a name used below, which is what is making Travis fail. 
maybe:
`from tensorflow_probability.python.bijectors import shift as shift_bijector`
Also please put a single import per line.

Comment (giovp) [reply tensorflow_probability/python/bijectors/masked_autoregressive.py]: Sorry for late reply. Seems like tests are still failing.

**Comment by giovp** [comment]
Hi @srvasude, I removed `affine_scalar` also from real_nvp but tests are failing there as well. 
Just want to mention that using the `Shift` and `Scale` bijectors as per #839 still seems to not have trainable variables. Any suggestion on this?
Thanks a lot in advance.

**Comment by srvasude** [comment]
Hi apologies for the delay on this,
   I added an option for `Scale` to take in `log_scale` now, so please try that (hoping that will fix the tests).

**Comment by giovp** [comment]
Eventually I re implemented by subclassing the bijector class, but this I believe should do the job. I see that it was changed already in master, so this PR can be closed I guess, right?

Thank you for having a look anyway!

**Comment by brianwa84** [comment]
Can you merge masked_autoregressive and rebase? The change to real_nvp is still relevant, if you want to go ahead w/ it.

**Comment by brianwa84** [comment]
You could also update masked_autoregressive to use the `log_scale` parameter to `tfb.Scale`, if you don't mind.

**Comment by giovp** [comment]
Hi @brianwa84 , added log_scale to both autoregressive and realnvp but some tests are still failing :( sorry for that but not sure how to address them.

Thank you for looking into this.

**Review by brianwa84 — APPROVED** [review]
will get this pulled in. thanks!

**Pull request merged**